installation.md

```diff
@@ -357,6 +357,9 @@ compile_commands.json file by recording a complete build.
 
 For a `make`-based build, you can run `make clean; bear -- make` to generate the
 file (and run a clean build!).
+
+On Windows, a tool similar to Bear called [compiledb](https://github.com/nickdiego/compiledb)
```

Comment:
This tool doesn't seem to be actively maintained, the last commit was 5 years ago...

Reply:
That's a good point.

Do you know of any other tools we could recommend to Windows users who are using `make` or other build systems without explicit `compile_commands.json` generation support?

Reply:
I don’t have a better suggestion. I guess it's still worth mentioning (assuming it still works today). It's better than having nothing.

Reply:
Though I haven't tried it myself (and I don't have a Windows environment to do so), users that I've recommended it to have reported it working.

Reply:
Then I have no further concerns.